frontend/src/metabase/api/user.ts

```diff
@@ -0,0 +1,68 @@
+import type {
+  CreateUserResponse,
+  CreateUserRequest,
+  ListUsersRequest,
+  ListUsersResponse,
+  UpdatePasswordRequest,
+  UserId,
+} from "metabase-types/api";
+
+import { Api } from "./api";
+// import { listTag, invalidateTags } from "./tags";
```

Comment:
Remove the commented out code.

```suggestion
```

Reply:
We'll need these for tags which are not implemented yet (see "FIXME" comments below)